commcare_connect/data_export/views.py

```
@@ -206,6 +215,41 @@ def get_queryset(self, request, opp_id):
+        many = isinstance(data, list)
+        if not many:
+            data = [data]
+
+        instances = []
+        for item in data:
+            user = item.pop("username", None)
+            if user:
+                user = User.objects.get(username=item.get("username"))
+            item["user"] = user
+            id = item.pop("id", None)
+            obj, created = LabsRecord.objects.update_or_create(defaults=item, **{"id": id})
```

Ruff (0.14.4)

Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)